package_control/package_cleanup.py

```diff
@@ -150,6 +150,11 @@ def run(self):
             if not os.path.isdir(package_dir):
                 continue
 
+            # Skipping user package
+            if package_name == "User":
+                console_write("Skip cleanup " + package_dir)
```

Comment:
Not sure whether it's worth a message in console as it is always the User package which is skipped. This is nothing to worry or pay attention to in some way.